src/platform_impl/linux/window.rs

```diff
@@ -152,8 +152,10 @@ impl Window {
     // https://docs.gtk.org/gtk3/drawing-model.html
     window.set_app_paintable(true);
     let widget = window.upcast_ref::<gtk::Widget>();
-    unsafe {
-      gtk::ffi::gtk_widget_set_double_buffered(widget.to_glib_none().0, 0);
+    if !event_loop_window_target.is_wayland() {
```

Comment:
should probably leave a TODO comment, or is this not needed on wayland?

Reply:
If it's just a window then it's not needed. For GL painting, I haven't done testing it yet. So we could just leave it.